mongodbatlas/private_endpoints.go

```diff
@@ -94,6 +100,17 @@ type GCPEndpoint struct {
 	ServiceAttachmentName string `json:"serviceAttachmentName,omitempty"` // Unique alphanumeric and special character strings that identify the service attachment associated with the endpoint.
 }
 
+// PrivateEndpointServerlessConnection represents MongoDB Private Endpoint Connection.
+type PrivateServerlessEndpointConnection struct {
+	ID                      string      `json:"_id,omitempty"` // Unique identifier of the Serverless PrivateLink Service.
+	CloudProviderEndpointID string      `json:"cloudProviderEndpointId,omitempty"`
+	Comment                 string      `json:"comment,omitempty"`
+	EndpointServiceName     string      `json:"endpointServiceName,omitempty"` // Name of the PrivateLink endpoint service in AWS. Returns null while the endpoint service is being created.
+	ErrorMessage            interface{} `json:"errorMessage,omitempty"`        // Error message pertaining to the AWS Service Connect. Returns null if there are no errors.
```

Comment:
why interface? the documentation specifies this is a string, is this because it says null?, ommitempty would turn null into an empty string which should be tter than type assertions

Reply:
Corrected to a string type